doc/extending/extending_theano.txt

```diff
@@ -791,7 +791,7 @@ implementation.
 It takes an optional :func:`infer_shape` parameter that must have this
 signature:
 
-.. code-block:: python
+.. code-block::
```

Comment:
```suggestion
.. code-block:: none
```
The tests use Sphinx 1.5.1.
Omitting the language is a Sphinx >2.0 feature: https://github.com/sphinx-doc/sphinx/blob/v2.3.1/CHANGES#L610-L611.